### ISSUE TITLE
MySQL server has gone away

### ISSUE THREAD

**Issue by sujoy84** [AshamaneProject/AshamaneCore#805]
**Description:**
Applying update "2018_04_09_01_world_localesESes.sql" 'F30627C'...
ERROR 2006 (HY000) at line 4: MySQL server has gone away

**Usefull Wowhead links:**

**Comment by Neredun** [comment]
i didn't have any problem on the applying of this update.

**Comment by sujoy84** [comment]
I have tried the following tricks:
wait_timeout=28800
interactive_timeout = 28800
max_allowed_packet=32M
Still the problem exists.

**Comment by lorac2k14** [comment]
@sujoy84 
I have my " max_allowed_packet= " 150M 
I was having similar issue until I upped the packets to 150M and for the most part it takes a very heavy packet to make MySQL go away for me. Which build are you running? 5.6.x or 5.7.x?
5.7.x builds are picky i find best luck with 5.6.x builds.
I hope this helps.

Side note:
I was able to run the full updates without MySQL going away. 
ill keep an eye on this thread in case you are still stuck.
Also remember you must restart the SQL server for new configs to take affect in case you were not aware of this.
